src/seq_mv/csr_matvec_device.c

```diff
@@ -13,7 +13,6 @@
 
 #include "seq_mv.h"
 #include "_hypre_utilities.hpp"
-#include "seq_mv.hpp"
```

Comment:
@waynemitchell Removing this line broke the GPU build with CUDA 11. What's the reason of removing it? for sycl? My reviewing did not catch this apparently.

Reply:
Yes, this broke our builds as well.  Thanks for catching it, @liruipeng!